src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -55,6 +55,11 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
                 break;
         }
 
+        if (options.Frontend.ConsoleLogHistoryLimit <= 0)
+        {
+            errorMessages.Add("ConsoleLogHistoryLimit must be greater than zero.");
```

Comment:
Add this name to DashboardConfigNames and use the ConfigKey property as the name. See other error messages in this type for examples.